operator-pipeline-images/operatorcert/tekton.py

```diff
@@ -85,7 +86,7 @@ class PipelineRun:
     # Markdown TaskRun template
     TASKRUN_TEMPLATE = "| {icon} | {name} | {start_time} | {duration} |"
 
-    def __init__(self, obj: dict, taskruns: list[TaskRun]) -> None:
+    def __init__(self, obj: dict, taskruns: Any) -> None:
```

Comment:
Why is this needed?

Reply:
I was getting type errors because of that, but It may be because of the python lower version I am using.